web-app/src/pages/tournament/TournamentsList.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <grid-list-mobile
+      :data="data"
+      :columns="columns"
+      :filter="filter"
+      :expanded="expanded"
+      :pagination="pagination"
+    />
+    <grid-list-desktop
+      :data="data"
+      :columns="columns"
+      :filter="filter"
+      :expanded="expanded"
+      :pagination="pagination"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator';
+import GridListMobile from '../../components/tournament/GridListMobile.vue';
+import GridListDesktop from '../../components/tournament/GridListDesktop.vue';
+@Component({
+  components: { GridListMobile, GridListDesktop },
+})
+export default class TournamentsList extends Vue {
```

Comment:
```suggestion
import GridListDesktop from '../../components/tournament/GridListDesktop.vue';

@Component({
  components: { GridListMobile, GridListDesktop },
})
export default class TournamentList extends Vue {
```
Should be `TournamentList`. This same in file name